languages/python/exercises/concept/lists/.docs/instructions.md

```diff
@@ -0,0 +1,151 @@
+## Instructions
+
+As a magician-to-be, Elyse needs to practice some basics. She has a stack of cards that she wants to manipulate. In this exercise we'll use lists to help Elyse perform her card tricks.
+
+[//]: # (Creating Lists)
```

Comment:
What does this do :O

Reply:
Looks like it makes a numbered list but at header size!  At least that's what appears in the doc preview.

Reply:
I love the narrative.  Great idea to use a "hand" & magic tricks!

Reply:
We'll want to add in relevant links to documentation on `lists` -- but we can do that later (_I can help with that_).